docs/how-to/deploy_cups_sdcore.md

```diff
@@ -0,0 +1,217 @@
+# Deploy SD-Core with CUPS
+
+This guide covers how to install a multi-node sdcore with separation of user and control plane traffic.
+
+### Requirements
+
+You will need two kubernetes clusters as follows.  In this how-to, we cover installation and configuration of MicroK8s, however, any compatible Kubernetes cluster can be used.
+- `SD-Core Control Plane`, where the full set of control plane functions will be placed (AMF, SMF, etc)
+- `SD-Core User Plane`, where the User Plane Function (UPF) will be placed
+
+A separate machine is required for the Juju controller.  While it is possible to install Juju on the same host as the SD-Core Control Plane, it is not recommended to do so.  [`TODO:` note on metallb for juju controller]
+
+In the `SD-Core User Plane` cluster, the following subnets must be present:
+- `access`, where the UPF listens for incoming sessions from the gNB
+- `core`, where the UPF send out traffic from the gNB towards the internet
+- `management`, for Juju and control plane traffic.  Must also have access to the internet for downloading container images, charms, or other updates as needed
```

Comment:
I think we are getting to the point where it might make sense to take the decision if we rename the networks or not. I personally think we should, but it might make sense to get the point of view from upstream.

@gruyaume any thoughts on this?

Reply:
I generally agree, this is more than a documentation effort though, we have to make changes in config in the UPF. We may want to keep this out of scope of this documentation change and create a new ticket that would include:
- Change in the UPF
- Change in this doc

Reply:
As this is larger than just documentation, we are not going to address this here.